examples/bedrock-agentcore-multi-agent/config.yaml

```
@@ -0,0 +1,250 @@
+  # Billing Agent - handles payment and subscription queries
+  - id: billing-agent
+    bedrock:agentcore:BILLING_AGENT_ID
+    config:
+      agentId: BILLING_AGENT_ID
+      agentAliasId: BILLING_ALIAS_ID
+      region: us-east-1
+      enableTrace: true
+      memoryConfig:
+        type: long-term
+        enabled: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)